users/JackPerdue/00_README.txt

```diff
@@ -0,0 +1,31 @@
+
+Being a clueless git with git, I've uploaded our successfully built easyconfigs to:
```

Comment:
have you looked at http://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html ?

It's meant exactly for clueless gits... ;-)